cluster/ci/config/prow/openshift/ci-operator/stage.yaml

```diff
@@ -0,0 +1,28 @@
+kind: List
+apiVersion: v1
+items:
+- kind: Role
+  apiVersion: authorization.openshift.io/v1
+  metadata:
+    name: ci-stg-user
+  rules:
```

Comment:
Do we need create on events here, too?

Reply:
`ci-operator` will use its own service account. This role is for the individual user accounts, which should interface with the namespace exclusively by creating pods generated by `mkpod`.

The reason ConfigMaps are included is to allow template authors (or adventurous users) to create their own template CM and change the job to use it (which have to has to be in the same namespace per kube's restrictions).

Reply:
Of course, that makes sense